homework/js-exercises/Time/showCurrentTime.js

```diff
@@ -0,0 +1,26 @@
+const updateTime = () => {
+    if (document.querySelector('h1') !== null) {
+        lastTime = document.querySelector('h1');
```

Comment:
You should always use a variable keyword (var, const, let) when defining a variable.

Reply:
also, no need for this assignment you could just 
```suggestion
        document.querySelector('h1').remove();
```